zerver/lib/markdown/fenced_code.py

```diff
@@ -189,6 +193,30 @@ def generic_handler(
         return TexHandler(processor, output, fence)
     elif lang == "spoiler":
         return SpoilerHandler(processor, output, fence, header)
+    elif lang in (
+        "blockdiag",
+        "seqdiag",
+        "nwdiag",
+        "actdiag",
+        "mermaid",
+        "plantuml",
+        "svgbob",
+        "packetdiag",
+        "rackdiag",
+        "graphviz",
+        "erd",
+        "excalidraw",
+        "vega",
+        "vegalite",
+        "ditaa",
+        "nomnoml",
+        "bpmn",
+        "bytefield",
+        "wavedrom",
+        "c4plantuml",
+        "umlet",
+    ):
```

Comment:
Is there a way we can get this list from kroki as a data file, or do we just have to hardcode it?

Reply:
Let's at least extract it to a variable at the top of the file `KROKI_SUPPORTED_DIAGRAM_LANGUAGES`.

Reply:
I'll try to generate a file but if not I'll create a variable in the top of the file

Reply:
We don’t need this list at all; we can accept ```` ```kroki anything ```` and pass it along.